src/WebJobs.Extensions.Http/HttpWebJobsStartup.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Hosting;
+using Microsoft.Extensions.Hosting;
+
+[assembly:WebJobsStartup(typeof(HttpWebJobsStartup))]
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    /// <summary>
+    /// Enable dynamic HTTP registration against WebJobs 
+    /// </summary>
+    public class HttpWebJobsStartup : IWebJobsStartup
```

Comment:
So this will auto-register the http extension with default options if http extension is registered? Is the storage extension and ServiceBus extension also already doing this? I looked and didn't see them doing so.